include/wil/coroutine.h

```diff
@@ -151,7 +151,7 @@
 #if defined(_RESUMABLE_FUNCTIONS_SUPPORTED)

```

Comment:
Not your change, and I'm not necessarily going to suggest touching it now, but this feels backwards. I'd think we'd want to check for the standard impl first, then fall back to experimental

Reply:
That's a good point. @oldnewthing do you recall if there was a reason for this ordering or is it arbitrary?